python/MooseDocs/test/extensions/test_appsyntax.py

```diff
@@ -156,7 +156,7 @@ def testLatex(self):
         # This size should correspond to the total number of parameters for
         # Diffusion (Required + Optional + Advanced + Tagging) + 1
         # (corresponding to 'type')
-        self.assertSize(res, 18)
+        self.assertSize(res, 19)
```

Comment:
doco not built rn. what did you add to diffusion?

Reply:
`matrix_only` was made non-private